assembl/static2/js/app/main.jsx

```diff
@@ -4,17 +4,13 @@ import * as React from 'react';
 import { connect } from 'react-redux';
 
 import { getCurrentPhaseIdentifier, type Timeline } from './utils/timeline';
-import { addRedirectionToV1 } from './actions/phaseActions';
 import Navbar from './components/navbar/navbar';
 import Footer from './components/common/footer';
 
-type Debate = { debateData: { timeline: Timeline } };
-
 type Props = {
-  debate: Debate,
+  timeline: Timeline,
```

Comment:
I think you could (should?) use the type from flow (generate it with `apollo-codegen` (see `flow/README` for details)